src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -1285,6 +1285,17 @@ def escape_quotes(unescaped: str) -> str:
     return unescaped.replace(DOUBLE_QUOTE, DOUBLE_QUOTE + DOUBLE_QUOTE)
 
 
+def uppercase_and_enquote_if_not_quoted(string):
```

Comment:
How is this function different than `quote_name`?

Reply:
no actually they are the same.. I removed this function and reused `quote_name`